### PR TITLE
Pass all select cols to prediction result table

### PR DESCRIPTION
**Pull request by Yancey0623**
fixed. #2244 

TODO: implement this feature on PAI.

### CODE REVIEW

File: python/sqlflow_submitter/db.py [hunk 1]
```diff
@@ -226,10 +247,11 @@ def reader():
                     feature = read_feature(row[field_names.index(name)],
                                            feature_specs[name], name)
                     features.append(feature)
+
                 if label_idx is None:
-                    yield (tuple(features), )
+                    yield list(row), tuple(features), None
```
Comment (typhoonzero) [inline python/sqlflow_submitter/db.py]: Yield one more duplicated data may not be efficient. Can we pass the full list of selected field to `db_generator`'s `feature_column_names` argument, then form a feature data just before call `predict`?

Comment (Yancey0623) [reply python/sqlflow_submitter/db.py]: Update as the following logic:
``` python
gen = db.generator()
for row, label in gen():
    features = db.read_features_from_row(row)
    predict(features, label)
```

File: python/sqlflow_submitter/tensorflow/input_fn.py [hunk 2]
```diff
@@ -53,6 +53,20 @@ def get_dtype(type_str):
         raise TypeError("not supported dtype: %s" % type_str)
 
 
+def tf_generator(gen, selected_cols, feature_column_names, feature_metas):
```
Comment (typhoonzero) [inline python/sqlflow_submitter/tensorflow/input_fn.py:56]: The name `tf_generator` is not telling what it does.

**Review by typhoonzero — APPROVED** [review]
LGTM with a naming comment

File: python/sqlflow_submitter/tensorflow/input_fn.py (hunk 2 above)
Comment (Yancey0623) [reply python/sqlflow_submitter/tensorflow/input_fn.py:56]: I will rename it in the next PR.

**Pull request merged**